challenges/2021_cloud_base_height/04_eval_pytorch.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>I think htis is the confusion matrix for a specific layer. It might need a bit more explanation of what this actually means, also maybe doesn't need to be so big.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
I added some small explaination to what I have done, I don't think the plot is terribly useful, but I was curious to see at the time